hc/blogs/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils import timezone
+from taggit.models import Tag
+from taggit.managers import TaggableManager
+
+class Post(models.Model):
```

Comment:
This class name is a little confusing. Please use a more descriptive name (something related to blog.)